src/new-hooks/components/paging-dots.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import _ from 'lodash';
```

Comment:
Just a note as we keep developing this, we don't want to import all of lodash as it is a pretty big package to add to our bundle. So we should only import the methods we are using

Reply:
it looks like this `import slice from 'lodash.slice';`

Reply:
Yeah gotcha 👍. I am going for this for now because I am quite unsure on things I would be messing around here. I will update them all pretty soon when I start to clean things up.

Reply:
totally! Just wanted to comment on it so I don't forget